models/Mentor.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const mentorSchema = Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  phone_number: {
```

Comment:
no snake case but camel case eg all should be phoneNumber instead of phone_number